ui/src/app/cohort-search/demographics/demographics.component.spec.ts

```diff
@@ -49,12 +48,6 @@ describe('DemographicsComponent', () => {
         {provide: NgRedux, useValue: mockReduxInst},
         {provide: CohortBuilderService, useValue: {}},
         {provide: CohortSearchActions, useValue: new MockActions()},
-        {
-          provide: ActivatedRoute,
-          useValue: {
-            snapshot: {data: {workspace: {cdrVersionId: '1'}}}
-          }
-        },
       ],
     })
```

Comment:
Do you need to push to the `urlParamsStore` here?

Reply:
I'm curious about this too...

Reply:
It's not necessary in all cases, since these tests don't always hit the code paths that actually read this data. Since this particular test code will be going away soon, I was just aiming to get things passing for now.